tasks/projects.yml

```diff
@@ -231,7 +231,6 @@
     floating_ips: "{{ quotas.floating_ips | default(omit) }}"
     floatingip: "{{ quotas.floatingip | default(omit) }}"
     gigabytes: "{{ quotas.gigabytes | default(omit) }}"
-    gigabytes_lvm: "{{ quotas.gigabytes_lvm | default(omit) }}"
```

Comment:
It looks like they have changed the format of the parameters for volume type quotas. `gigabytes_lvm` becomes:

```
gigabytes_types:
      gigabytes_lvm: "{{ item.gigabytes_lvm }}"
```

I suggest we keep support for `gigabytes_lvm`, and add `gigabytes_types` as a new parameter. With `default(omit)` it should continue to work.

Reply:
To be clear, I'm suggesting adding this:

```
gigabytes_types: "{{ item.gigabytes_types }}"
```

Reply:
They did not. They've had misleading docs - which lead to a broken "support" in this role - details in #36

Reply:
Ack